cypher/cypher-docs/src/docs/dev/syntax/comparison.asciidoc

```diff
@@ -63,7 +63,26 @@ The following points give some details on how the comparison is performed.
  A leap year occurs on a year with an ordinal number divisible by `4`, that is not divisible by `100`, unless it divisible by `400`.
  This means that over `400` years there are `((365 * 4 + 1) * 25 - 1) * 4 + 1 = 146097` days, which means an average of `365.2425` days per year.].
 * Comparing for ordering when one argument is `null` (e.g. `null < 3` is `null`).
-
+* *Ordering* of values with *different* types:
+ ** The ordering is, in ascending order, defined according to the following list:
+  *** `MAP` types
+   **** <<cypher-literal-maps, Regular maps>>
+   **** <<cypher-map-projection, `NODE` maps>>
+   **** <<cypher-map-projection, `RELATIONSHIP` maps>>
+  *** <<cypher-lists, `LIST`>>
+  *** <<cypher-pattern-path-variables, `PATH`>>
+  *** <<cypher-temporal-introduction, `DATETIME`>>
+  *** <<cypher-temporal-introduction, `LOCALDATETIME`>>
+  *** <<cypher-temporal-introduction, `DATE`>>
+  *** <<cypher-temporal-introduction, `TIME`>>
+  *** <<cypher-temporal-introduction, `LOCALTIME`>>
+  *** <<cypher-temporal-introduction, `DURATION`>>
+  *** <<cypher-expressions-general, `STRING`>>
+  *** <<cypher-expressions-general, `BOOLEAN`>>
+  *** <<cypher-expressions-general, `NUMBER`>>
+ ** The value `NULL` is considered larger than any value.
+ ** For the container types, elements of the containers use orderability to determine the order between them.
```

Comment:
I don't think "orderability" is a phrase introduced anywhere in our documentation. Consider phrasing it more like "the elements of the container type are compared pairwise for ordering and thus determine the ordering of two container types ..."

I think this is orthogonal to the change with `*Ordering* of values with *different* types`. Ordering of container types could be its own bullet point with sub-bullet-points for lists and maps. We also seem to call them [Composite](https://neo4j.com/docs/cypher-manual/current/syntax/values/#composite-types) types and not container types.